application/cms/models.py

```diff
@@ -1011,6 +1011,7 @@ class Topic(db.Model):
     title = db.Column(db.String(255), nullable=False)
     description = db.Column(db.Text, nullable=True)  # a sentence below topic heading on homepage
     additional_description = db.Column(db.TEXT, nullable=True)  # short paragraph displayed on topic page
+    meta_description = db.Column(db.TEXT, nullable=True)  # sentence or two for SEO/open graph description
```

Comment:
Comment could be something like "sentence or two for search engines and social sharing"? Not sure everyone knows what open graph is, given that it's not exactly been massively successful (outside of Facebook).